more/free-programming-cheatsheets.md

```diff
@@ -149,7 +149,7 @@
 
 ### Java
 
-* [Java](https://programmingwithmosh.com/wp-content/uploads/2019/07/Java-Cheat-Sheet.pdf) (PDF)
+* [Java](https://programmingwithmosh.com/wp-content/uploads/2019/07/Java-Cheat-Sheet.pdf) - Moshfegh Hamedani (PDF)
```

Comment:
Maybe adding `(Programming with Mosh)` makes more sense to create a standard.

Reply:
course listings need the extra context because the affordances vary by platform. Not so true for books - we don't list the publisher unless its a corporate author.

Reply:
good questions, though, it helps all of us do a better job reviewing.